lib/fetch/formdata.js

```diff
@@ -0,0 +1,217 @@
+'use strict'
+
+const { Blob } = require('buffer')
+const { kState } = require('./symbols')
+const { File } = require('./file')
+const { HTMLFormElement } = require('./util')
+
+class FormData {
```

Comment:
We could, and probably should, bring this into core also.

Reply:
ditto with `Headers`

Reply:
@Ethan-Arrowood has already been working in that direction https://github.com/nodejs/node/pull/38986

Reply:
Yes, we based our work on his. I believe he discontinued his efforts in favor of including it through undici.